Makefile

```
@@ -20,3 +17,23 @@ $(OUT): $(SRC)
 
 clean:
 	rm -f $(OUT)
+
+install: $(OUT)
+	cp ./port-lock /usr/bin/
+	rm -f /etc/systemd/system/port-mgr.service
+	cp ./port-mgr.service /etc/systemd/system/
+	rm -f /usr/bin/port-lock.sh
+	cp ./port-lock.sh /usr/bin/
+	systemctl daemon-reload
+	systemctl enable port-mgr.service
+	systemctl start port-mgr.service
+
+uninstall:
+	systemctl stop port-mgr.service
+	systemctl disable port-mgr.service
+	systemctl daemon-reload
+	rm -f /etc/systemd/system/port-mgr.service
```

```suggestion
	rm -f /etc/systemd/system/port-mgr.service
	systemctl daemon-reload
```

Switch the order since we want to reload all available service files after we have removed `port-mgr.service`.